Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SD-2-B.yml

```diff
@@ -20,19 +20,19 @@ title:
 
 #Describe the benefit that is achieved by implementing this activity
 benefit: 
-  Risk of leaking production secrets is mitigated by removing any manual interactions during deployment.
+  Risk of leaking production secrets is reduced by removing them from source code files.
```

Comment:
Production secrets are not in source code files already on level 1, what's the idea here?

Reply:
To me, in level 1, they can be in configuration files, but not accessible to developers. For instance, by using encrypted secrets (like in .Net). At level 2, you want to get rid of secrets in config files.

Reply:
I see, (while I would allow both on level 1 - crypto might also not get trivial) + on level 1, the deployment process is not necessarily automated. + the idea was to cover more than the source code, rather the whole deployment process. But maybe it's just me, could be discussed in the call.

Reply:
I'm not sure I understand your comment here: the deployment process itself is in stream A (and I agree, is should be about description, automation, ...) , while here we're talking about stream B = secret management. The latter should IMO be about where do you store secrets, how go get them into the application, and what's their lifecycle. No ?